dndserver/handlers/lobby.py

```diff
@@ -74,3 +84,28 @@ def open_map_select(ctx, msg):
     req.ParseFromString(msg)
     res = SS2C_OPEN_LOBBY_MAP_RES()
     return res
+
+
+def auto_match(ctx, msg):
+    req = SC2S_AUTO_MATCH_REG_REQ()
+    req.ParseFromString(msg)
+    party = get_party(account_id=sessions[ctx.transport].account.id)
+    for user in party.players:
+        transport, session = get_user(account_id=user.account.id)
+        if req.mode == 1:
+            if len(party.players) > 1:
+                matchteam = SS2C_AUTO_MATCH_REG_TEAM_NOT(result=1, mode=req.mode)
```

Comment:
I think it would be better to change the results to the enum as well
```suggestion
                matchteam = SS2C_AUTO_MATCH_REG_TEAM_NOT(result=pc.SUCCESS, mode=req.mode)
```